src/Scripting/Compiler/BytecodeEmitter.cpp

```
@@ -0,0 +1,911 @@
+#include "BytecodeEmitter.hpp"
+#include "BytecodeEmitter.hpp"
```

Clang (14.0.6)

[warning] duplicate include

(readability-duplicate-include)